app/ports/tests/test_buildhistory.py

```diff
@@ -0,0 +1,45 @@
+import os
+
+from django.test import TestCase
+from django.urls import reverse
+
+from ports.models import Dependency, Port, BuildHistory, Builder
+from MacPorts.config import TEST_PORTINDEX_JSON, BUILDS_FETCHED_COUNT
+
+
+class TestDependencies(TestCase):
```

Comment:
Does this not need be a `TransactionTestCase` with `reset_sequences` enabled?

Reply:
This does not need, but for consistency, I would use `TransactionTestCase` here also.

Reply:
No, I didn't mean for consistency across the project but really the test itself.